manifests/config.pp

```diff
@@ -35,7 +35,10 @@
     })
   }
 
-  ensure_resource('file', $::jenkins::localstatedir, $dir_params)
+  if $::jenkins::manage_localstatedir {
+    ensure_resource('file', $::jenkins::localstatedir, $dir_params)
+  }
+
```

Comment:
@jhoblitt In my case, my localstatedir already contains those directors and they're managed by the user and group that I'm passing in as the jenkins user and group, so it wouldn't change much. I was more just worried about having the option of defining localstatedir as a symlink.

It might be a good idea to toggle all the directory resources to support other possibilities.